### PR TITLE
Paas 1510 move backup to jelastic packages

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-1510

Short description:

Good luck :trollface:

### CODE REVIEW

File: mixins/common.yml [hunk 1]
```diff
@@ -227,3 +227,21 @@ actions:
...
+        ## [${this.logAction}] - 1/5
+        sudo yum install -y python36 python36-pip
+        sudo python3 -m pip install --upgrade pip
+        sudo python3 -m pip install --upgrade boto3 "azure==4.0.0" adal
+        [ -d jelastic_backup ] || mkdir jelastic_backup
+        cd jelastic_backup
+        for file in backrest.py lib_aws.py lib_azure.py revisionNode.py; do
+          curl -fLSso $file ${globals.repoRootUrl}/assets/common/$file
+        curl -fLSso elasticsearch.py ${globals.repoRootUrl}/assets/elasticsearch/elasticsearch.py
+        done
```
Comment (laurentfufu) [inline mixins/common.yml:243]: something is weird here

File: packages/common/auto_backup.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+type: install
+version: 1.5.2
+name: Jahia Cloud - Scheduled backup
+logo: /images/jahia-logo-70x70.png
+id: jahia-cloud-scheduled-backup
+description:
+  short: Jahia Cloud - Scheduled backup
+  text: blabla bla
```
Comment (laurentfufu) [inline packages/common/auto_backup.yml:9]: maybe we can remove this... um... _placeholder_ :trollface:

**Review by laurentfufu — COMMENTED** [review]
(review not finished yet, very long :trollface: )

File: packages/common/auto_backup.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+type: install
+version: 1.5.2
+name: Jahia Cloud - Scheduled backup
+logo: /images/jahia-logo-70x70.png
```
Comment (laurentfufu) [inline packages/common/auto_backup.yml:5]: bad path

File: packages/common/auto_backup_control.yml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+actions:
+  initVars:
+    - script: |-
+        var newUrl = "${baseUrl}".split('/')
+        newUrl.pop();
+        newUrl.pop();
+        return {result: 0, "backup_url": newUrl.join('/')};
+    - setGlobals:
+        backup_url: ${response.backup_url}
```
Comment (laurentfufu) [inline packages/common/auto_backup_control.yml:30]: don't see where this global is used

File: packages/common/listbackup.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+version: 1.5
+name: Jahia - List backup in bucket
+logo: ../../assets/common/jahia-logo-70x70.png
+id: jahia-list-from-bucket
+
+globals:
+  logAction: "List"
+
+mixins:
+  - mixins/common.yml
```
Comment (laurentfufu) [inline packages/common/listbackup.yml:12]: bad path

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: mixins/common.yml (hunk 1 above)
Comment (adoux) [reply mixins/common.yml:243]: :man_shrugging: 
one of the files is in "elasticsearch" subfolder where all the others are in "common", that's why

File: packages/common/auto_backup.yml (hunk 2 above)
Comment (adoux) [reply packages/common/auto_backup.yml:9]: Damn I did it earlier but the update must have been lost in the process :man_facepalming:

File: packages/common/auto_backup.yml (hunk 3 above)
Comment (adoux) [reply packages/common/auto_backup.yml:5]: Good catch!

File: packages/common/auto_backup_control.yml (hunk 4 above)
Comment (adoux) [reply packages/common/auto_backup_control.yml:30]: Indeed, I removed that part!

File: packages/common/listbackup.yml (hunk 5 above)
Comment (adoux) [reply packages/common/listbackup.yml:12]: Good catch!

File: mixins/common.yml (hunk 1 above)
Comment (laurentfufu) [reply mixins/common.yml:243]: but maybe download only once the "elasticsearch" one will be enough ? :trollface: 
indentation is wrong but the curl for "elasticsearch.py" is down for each iteration in the for loop, so maybe shit line line 242 and 243 is the thing to do

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: mixins/common.yml (hunk 1 above)
Comment (adoux) [reply mixins/common.yml:243]: Oh damn, I'm King Dumb :man_facepalming:

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**